src/test/java/org/minions/devfund/mauricioz/linkedList/RemoveMaximumValuesTest.java

```diff
@@ -0,0 +1,219 @@
+
+        if (var1.size() > LISTSIZE1) {
+            Assert.fail("removeMaximumValues does not remove enough elements from LinkedList when number of elements "
+                    + "to remove is less than size of input");
+        }
+
+        if (var1.contains("kangaroo") || var1.contains("koala")) {
+            Assert.fail("removeMaximumValues does not remove correct elements from LinkedList when number of "
+                    + "elements to remove is less than size of input");
+        }
+
+        Assert.assertTrue("removeMaximumValues does not have elements in correct order when number of "
+                + "elements to remove is less than size of input", ((String) var1.get(0)).equals("doge"));
+        Assert.assertTrue("removeMaximumValues does not have elements in correct order when number of "
+                + "elements to remove is less than size of input", ((String) var1.get(1)).equals("cat"));
+        Assert.assertTrue("removeMaximumValues does not have elements in correct order when number of "
+                + "elements to remove is less than size of input", ((String) var1.get(2)).equals("bear"));
+    }
+
+    /**
+     * test.
+     */
+    @Test
+    public void testMaxValuesAppearMoreThanOnce() {
+        LinkedList var1 = new LinkedList();
+        var1.add("doge");
+        var1.add("kangaroo");
+        var1.add("cat");
+        var1.add("kangaroo");
+        var1.add("koala");
+        var1.add("bear");
+        var1.add("doge");
+        var1.add("koala");
+        var1.add("kangaroo");
+
+        try {
+            LinkedListUtils.removeMaximumValues(var1, 2);
+        } catch (Exception var3) {
+            Assert.fail("removeMaximumValues throws " + var3 + " when number of elements to remove is less than "
+                    + "size of input and maximum elements appear more than once");
+        }
+
+        if (var1.size() < LISTSIZE2) {
+            Assert.fail("removeMaximumValues removes too many elements from LinkedList when number of elements to "
+                    + "remove is less than size of input and maximum elements appear more than once");
+        }
+
+        if (var1.contains("kangaroo") || var1.contains("koala")) {
+            Assert.fail("removeMaximumValues does not remove all instances of max elements from LinkedList when "
+                    + "number of elements to remove is less than size of input and maximum elements appear more "
+                    + "than once");
+        }
+
+        if (var1.size() > LISTSIZE2) {
+            Assert.fail("removeMaximumValues does not remove enough elements from LinkedList when number of elements "
+                    + "to remove is less than size of input and maximum elements appear more than once");
+        }
+
+        Assert.assertTrue("removeMaximumValues does not have elements in correct order when number of "
+                + "elements to remove is less than size of input and maximum elements appear more than once",
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define a constant instead of duplicating this literal "elements to remove is less than size of input and maximum elements appear more than once" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)